COMPILING.md

```diff
@@ -145,3 +145,15 @@ Will build the file and make it available in `./Release/Jamulus.app`
 * Run `/path/to/qt/5.15.2/ios/bin/qmake -spec macx-xcode Jamulus.pro`
 * Open the generated .xcodeproject in XCode, check the Signing & Capabilities and configure a team
 * To run on a iOS device, build and run on the device, you'll have first trust your developer profile in device's Settings.
+
+## Android
```

Comment:
I don't know anything about Android development, but to refer explicitly to UI elements "the Projects tab" and menus, as well as `git clone` and `cd jamulus` seems a bit unnecessary/risky to me. Can we not bring it into line with the brevity of the other platforms?

When I originally did this doc, I removed a bunch of stuff I assumed experienced engineers/power users would see as noise, and we would be lumbered with checking in future...